pallets/loan/src/tests.rs

```diff
@@ -109,7 +109,7 @@ fn issue_loan() {
 	TestExternalitiesBuilder::default()
 		.build()
 		.execute_with(|| {
-			let owner: u64 = 1;
+			let owner: u64 = 10;
```

Comment:
For my education, could you explain this change? :)

Reply:
1 is admin account. 10 is non admin